src/Entity/Post.php

```diff
@@ -33,77 +30,57 @@
  * @author Javier Eguiluz <javier.eguiluz@gmail.com>
  * @author Yonel Ceruto <yonelceruto@gmail.com>
  */
+#[ORM\Entity(repositoryClass: PostRepository::class)]
+#[ORM\Table(name: 'symfony_demo_post')]
+#[UniqueEntity(fields: ['slug'], errorPath: 'title', message: 'post.slug_unique')]
 class Post
 {
-    /**
-     * @ORM\Id
-     * @ORM\GeneratedValue
-     * @ORM\Column(type="integer")
-     */
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
     private ?int $id = null;
 
-    /**
-     * @ORM\Column(type="string")
-     */
+    #[ORM\Column(type: 'string')]
     #[Assert\NotBlank]
     private ?string $title = null;
 
-    /**
-     * @ORM\Column(type="string")
-     */
+    #[ORM\Column(type: 'string')]
     private ?string $slug = null;
 
-    /**
-     * @ORM\Column(type="string")
-     */
+    #[ORM\Column(type: 'string')]
     #[
         Assert\NotBlank(message: 'post.blank_summary'),
         Assert\Length(max: 255)
     ]
     private ?string $summary = null;
 
-    /**
-     * @var string
-     *
-     * @ORM\Column(type="text")
-     */
+    #[ORM\Column(type: 'text')]
     #[
         Assert\NotBlank(message: 'post.blank_content'),
         Assert\Length(min: 10, minMessage: 'post.too_short_content')
     ]
     private ?string $content = null;
 
-    /**
-     * @ORM\Column(type="datetime")
-     */
+    #[ORM\Column(type: 'datetime')]
     private \DateTime $publishedAt;
 
-    /**
-     * @ORM\ManyToOne(targetEntity="App\Entity\User")
-     * @ORM\JoinColumn(nullable=false)
-     */
+    #[ORM\ManyToOne(targetEntity: User::class)]
+    #[ORM\JoinColumn(nullable: false)]
     private ?User $author = null;
 
     /**
      * @var Comment[]|Collection
-     *
-     * @ORM\OneToMany(
-     *      targetEntity="Comment",
-     *      mappedBy="post",
-     *      orphanRemoval=true,
-     *      cascade={"persist"}
-     * )
-     * @ORM\OrderBy({"publishedAt": "DESC"})
      */
+    #[ORM\OneToMany(targetEntity: Comment::class, mappedBy: 'post', orphanRemoval: true, cascade: ['persist'])]
+    #[ORM\OrderBy(['publishedAt' => 'DESC'])]
     private Collection $comments;
 
     /**
      * @var Tag[]|Collection
-     *
-     * @ORM\ManyToMany(targetEntity="App\Entity\Tag", cascade={"persist"})
-     * @ORM\JoinTable(name="symfony_demo_post_tag")
-     * @ORM\OrderBy({"name": "ASC"})
      */
+    #[ORM\OneToMany(targetEntity: Tag::class, mappedBy: 'post', cascade: ['persist'])]
```

Comment:
the bug is that you added `mappedBy: 'post'` here, which is wrong (and was not there in the annotation

Reply:
btw, the linter job gives you the error directly in the "lint doctrine entities" failure.

Reply:
I added the `mappedBy` property because otherwise I see this error:

![doctrine-error](https://user-images.githubusercontent.com/73419/166984483-1f6e782d-ad9f-46ff-ac60-e2bc2976ed4d.png)

Reply:
@javiereguiluz, You changed the relation type from ManyToMany to OneToMany.

```suggestion
    #[ORM\ManyToMany(targetEntity: Tag::class, cascade: ['persist'])]
```

Reply:
Yes!!! Super nice catch! That was it. Thanks a lot Oleg 🙇🙇  Fixed!